Software/vortex_ws/src/mission_planning/README.md

```diff
@@ -1 +1,173 @@
-mission planning Package
+# Mission Planning Package
+
+## Description
+
+It starts with the **planning layer** which contains **task manager**, it sends next mission to be executed to the execution layer based on the labeled camera feed.
```

Comment:
based on the labeled camera feed and mission priority and it can be breakdown to the following modules
I think this will be better description, what do you think

Reply:
Adding "and mission priority" will describe the function better, But I can't add "it can be breakdown to the following modules" since the next mentioned modules are not part of the task manager module, they are different modules in the main (mission planning module)